utilities/get_android_apks.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+from pathlib import Path
+
+from bs4 import BeautifulSoup
+import requests
+
+
+parser = argparse.ArgumentParser("Options for android apk downloader")
+
+parser.add_argument('--firefox-version', help="The firefox version to download")
+args = parser.parse_args()
+
+headers = {
+    'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36',
+    'Accept': 'application/json',
+    'Content-Type': 'application/json',
+}
+jobs_filter = ["signing-apk-fenix-debug", "signing-apk-fenix-android-test-debug"]
+download_ids = {}
+task_ids = {}
+path = Path().cwd()
+
+firefox_version = args.firefox_version.replace(".", "_")
+hg_url = requests.get(f"https://hg.mozilla.org/releases/mozilla-release/rev/FIREFOX-ANDROID_{firefox_version}_RELEASE").content
+
+soup = BeautifulSoup(hg_url, 'html.parser')
+
+treeherder_link = soup.find("a", string='default view')
+treeherder_link = treeherder_link.get('href')
+
+revision = treeherder_link.split('=')[-1]
+
+result_id_json = requests.get(f"https://treeherder.mozilla.org/api/project/mozilla-release/push/?full=true&count=10&revision={revision}", headers=headers).json()
+result_id = result_id_json.get("results")[0].get("id")
+
+fenix_jobs = requests.get("https://treeherder.mozilla.org/api/project/mozilla-release/jobs/?job_group_symbol=fenix-debug&count=100", headers=headers).json()
```

Comment:
Also, what is the `count=100` here?
Below it looks like we're looping over `fenix_jobs["results"]` so possibly a 100-iteration loop to find our needle-vs-haystack.
Might want to toss in a couple documentation strings for Future Ben.

Reply:
The `count=100` is for the pagination on the results. I need to load a few of the results from this call so this gives me like 100.